luaui/configs/keyboard_layouts.lua

```diff
@@ -76,7 +76,51 @@ scanToCode["dvorak"] = {
 	-- NEEDS CORRECTION ABOVE
 }
 
-local layouts = { 'qwerty', 'qwertz', 'azerty', 'dvorak' }
+scanToCode["de-neo"] = {
+	Q = "X",
+	W = "V",
+	E = "L",
+	R = "C",
+	T = "W",
+	Y = "K",
+	U = "H",
+	I = "G",
+	O = "F",
+	P = "Q",
+	A = "U",
+	S = "I",
+	D = "A",
+	F = "E",
+	G = "O",
+	H = "S",
+	J = "N",
+	K = "R",
+	L = "T",
+	Z = "Ü",
+	X = "Ö",
+	C = "Ä",
+	V = "P",
+	B = "Z",
+	N = "B",
+	M = "M",
+	[";"] = "d",
+	["'"] = "y",
+	[","] = ",",
```

Comment:
It's likely this one requires `["comma"]` instead.

You can test it by doing something like:

```
/unbindkeyset sc_comma
/bind sc_comma stop
```

Then hovering over the stop button in order menu and seeing what the tooltip shows

Reply:
If I put in the commands you listed, the tooltip of the stop order still shows 'O' (capital letter o), just as before. It doesn't change the tooltip.

Reply:
requires toggling the widget on and off to pick up the update